### ISSUE TITLE
Docker Daemon often stops responding

### ISSUE THREAD

**Issue by jamestenglish** [docker/for-win#949]
### Expected behavior
Running `docker ps` in PowerShell should return the state of the running containers
### Actual behavior
Intermittently 
`docker ps` starts returning:
```
Error response from daemon: Bad response from Docker engine
```

After this point the daemon is unreachable

The Docker settings window still shows Docker as running:
![image](https://user-images.githubusercontent.com/212868/28470309-3727f700-6dee-11e7-82e1-463613e8a899.png)

### Information

  - Diagnostic ID: EF9888B1-18C0-4038-872D-796EC16BF93C/2017-07-21_08-24-36
  - Windows 10 Enterprise, version 1703, build 15063.483
  - Docker version 17.06.0-ce-win19 (12801)

### Steps to reproduce the behavior
I am unable to reliably reproduce the behavior but this has been happening to me several times a day for the past few days. The only temporary remedy is to restart docker.

**Comment by gorbunovav** [comment]
this happens for us too on several computers, on both stable and edge versions

**Comment by jasonbivins** [comment]
Hi @jamestenglish @gorbunovav 
 Can you check the power management setting on your NIC for me, and disable it if active?
This is a longshot, but I've seen this setting do the weirdest things to programs, especially ones that are as network dependent as Docker. 

![image](https://user-images.githubusercontent.com/28021664/28644184-878c552e-7226-11e7-9631-d442162ad81f.png)

**Comment by jamestenglish** [comment]
@jasonbivins I tried that and am still seeing issues.

I think it may have to do with the resource allocation for the MobyVM. When I did a "Reset to factory defaults" things worked fine for several days. However I have containers that need more than the 2GB and CPU.

I have 12 cores and 16GB ram, when I bump the MobyVM Resources up to like 10 CPUs and 12GB I get docker daemon failures often. I've moved it down to 4 CPU and 6GB ram and the failures still happen but are much less frequent - about once a day rather than once an hour.

Is there some gotcha or limit w/ Hyper-V and the MobyVM I am running into w/ expanded resource allocation?
